tests/test_vcs.py

```diff
@@ -17,10 +17,26 @@
 from unittest import mock
 
 from fetchcode.vcs import fetch_via_vcs
+from fetchcode.vcs.pip._internal.vcs import vcs
 
 
-def test_fetch_with_git_url_returns_a_response():
-    url = 'git+http://github.com/jamesor/mongoose-versioner'
+def obtain(dest, url):
+    pass
```

Comment:
Just for my knowledge, what does this function do?

Reply:
It installs the given VCS Url's content at Destination.